frontend/src/components/videos/videos_index.jsx

```diff
@@ -5,6 +5,13 @@ import Loader from "react-loader-spinner";
 import VideosIndexItem from "./videos_index_item";
 import "./animate.css";
 
+import RightArrow from "./right-arrow.png";
+import LeftArrow from "./left-arrow.png";
+import Space from "./space-btn.png";
+
+const FIREBASE_VIDEO_URL =
```

Comment:
delete unused variable in file

Reply:
hmm, I use the variables in the file though..